### PR TITLE
Add detail wrapper component and add basic inv info to it

### PR DESCRIPTION
**Pull request by karelhala**
Screenshot from 2020-08-04 17-44-14

### CODE REVIEW

File: packages/inventory/src/components/detail/TopBar.js [hunk 1]
```diff
@@ -37,7 +42,21 @@ const TopBar = ({
...
                     loaded ? (
-                        <Title headingLevel="h1" size='2xl'>{ entity && entity.display_name }</Title>
+                        <Flex>
+                            <FlexItem>
+                                <Title headingLevel="h1" size='2xl'>{ entity && entity.display_name }</Title>
+                            </FlexItem>
+                            {
+                                showTags &&
+                                <FlexItem>
+                                    {loaded ?
```
Comment (Hyperkid123) [inline packages/inventory/src/components/detail/TopBar.js:52]: This condition does not make sense. `loaded` must always be `true` at this point.

**Comment by Hyperkid123** [comment]
@karelhala npm seems to have some issues atm (we have the same problem on multiple respositories)

**Comment by Hyperkid123** [comment]
@karelhala is this ready for review or do you plan to do some changes?

**Comment by karelhala** [comment]
@Hyperkid123 ready for review, I was just fixing small bugs here and there.

**Pull request merged**